README.md

```diff
@@ -1,66 +1,75 @@
 # starter-snake-python
 
-A simple [Battlesnake AI](http://battlesnake.io) written in Python. 
+A simple [Battlesnake AI](http://play.battlesnake.com) written in Python.
 
-Visit [https://github.com/battlesnakeio/community/blob/master/starter-snakes.md](https://github.com/battlesnakeio/community/blob/master/starter-snakes.md) for API documentation and instructions for running your AI.
+Visit [https://github.com/BattlesnakeOfficial/community/blob/master/starter-snakes.md](https://github.com/BattlesnakeOfficial/community/blob/master/starter-snakes.md) for API documentation and instructions for running your AI.
 
 This AI client uses the [bottle web framework](http://bottlepy.org/docs/dev/index.html) to serve requests and the [gunicorn web server](http://gunicorn.org/) for running bottle on Heroku. Dependencies are listed in [requirements.txt](requirements.txt).
 
 [![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy)
 
-#### You will need...
+## Prerequisites
 
-* a working Python 2.7 development environment ([getting started guide](http://hackercodex.com/guide/python-development-environment-on-mac-osx/))
-* experience [deploying Python apps to Heroku](https://devcenter.heroku.com/articles/getting-started-with-python#introduction)
-* [pip](https://pip.pypa.io/en/latest/installing.html) to install Python dependencies
+- a working Python 3.8 development environment ([getting started guide](http://hackercodex.com/guide/python-development-environment-on-mac-osx/))
```

Comment:
repo is not actually 3.8 ready yet, keep at 2.7 for now

Reply:
Oops, I cherry-picked this from a branch I was working on migrating to 3.8 😬